source/agora/consensus/protocol/Nominator.d

```diff
@@ -70,6 +74,12 @@ public extern (C++) class Nominator : SCPDriver
     /// The quorum set
     private SCPQuorumSetPtr[Hash] quorum_set;
 
+    /// Tracks unique incremental timer IDs
+    private ulong[TimerType] last_timer_id;
```

Comment:
That's an AA, no ?

```suggestion
    private ulong[TimerType.max + 1] last_timer_id;
```

To make it a static array ? I really wish DMD recognized this pattern.

Reply:
Good idea.